feature/org.wso2.carbon.identity.common.feature/pom.xml

```diff
@@ -43,6 +43,10 @@
             <artifactId>org.wso2.carbon.identity.event</artifactId>
             <version>${project.version}</version>
```

Comment:
Shall we remove the version from the dependency tag here and move it to the parent pom?